eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/utils/linkify/CompleteWebUrlLinkifying.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017 SchedJoules
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.schedjoules.eventdiscovery.framework.utils.linkify;
+
+import android.support.v4.text.util.LinkifyCompat;
+import android.text.Spannable;
+import android.text.util.Linkify;
+import android.util.Patterns;
+
+import com.schedjoules.eventdiscovery.framework.utils.spanned.AbstractSpannable;
+
+import java.util.regex.Matcher;
+
+
+/**
+ * A {@link Spannable} that linkifies web urls that starts with "http" or "Http" in the input {@link Spannable}.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class CompleteWebUrlLinkifying extends AbstractSpannable
```

Comment:
The name is not correct. It should be "Linkified" instead of "Linkifying", because it doesn't decorate behavior but properties, i.e. it's a "linkified Spannable". Maybe call it `HttpUrlLinkified`